ruapu.h

```diff
@@ -334,6 +335,54 @@ RUAPU_INSTCODE(xtheadmempair, 0xe0a1450b) // th.lwd a0,a0,(sp),#0,3
 RUAPU_INSTCODE(xtheadsync, 0x0180000b) // th.sync
 RUAPU_INSTCODE(xtheadvdot, 0x8000600b) // th.vmaqa.vv v0,v0,v0
 
+// RVV 1.0 support
+static void ruapu_rvv_assert(int cond) {
+    // unimp
+    if (!cond)
+        asm volatile(".align 2\n.word 0xc0001073" : : : );
```

Comment:
Isn't `.WORD` enough to represent align 2?

Reply:
Emm, the `.align` directive is required since compiler may generate `c.beqz` for `if(!cond)`.
Without this directive:
```
0000000000001e4c <ruapu_rvv_assert>:
    1e4c:	e119                	bnez	a0,1e52 <ruapu_rvv_assert+0x6>
    1e4e:	c0001073          	.word	0xc0001073
    1e52:	8082                	ret
```